src/graphics/gfx_enter_screen.cpp

```diff
@@ -143,7 +143,7 @@ static void initPlayers(Player_t tempPlayer[maxLocalPlayers])
 
 static void drawEnterScreen(Player_t tempPlayer[maxLocalPlayers])
 {
-    for(int A = 0; A < numPlayers; ++A)
+    for(int A = 0; A < maxLocalPlayers; ++A)
```

Comment:
Maybe better to do this one with the A < max && A < num pattern? Uncertain how the inner code works though.

Reply:
You right

Reply:
Done, now the check rely on both num players and max players